src/components/molecules/UiChip/UiChip.vue

```diff
@@ -134,20 +135,25 @@ if (buttonAttrs.value) {
         color,
         var(--color-chip-icon-background-active)
       )};
-
-    margin: functions.var($element + "-icon", margin, -2px);
+    --button-icon-margin: functions.var($element + "-icon", margin, calc(var(--space-2) * -1));
+    --button-rtl-icon-margin: #{functions.var($element + "-rtl-icon", margin, calc(var(--space-2) * -1))};
   }
 
   &__remove {
-    margin: functions.var($element + "-remove", margin, var(--space-2) var(--space-2) var(--space-2) var(--space-4));
+    position: relative;
+    align-self: flex-start;
+    margin: functions.var($element + "-remove", margin, var(--space-2));
+
+    &::before {
```

Comment:
@ArkadiuszKrzymienski WDYT about moving this part to `mixin.pointer-area` in the next PR?
```
@mixin pointer-area {
  position: relative;

  &::before {
    ...
  }
}
```
`position: relative` is necessary when you want to use `position: absolute` fore `before` pseudoelement. In this case UiButton has `position: relative` but we should add it anyway :)

Reply:
Sure 👍 It will be useful in the future 😊